benchopt/benchmark.py

```diff
@@ -220,12 +220,20 @@ def get_result_file(self, filename=None):
             if filename == 'all':
                 result_filename = all_result_files
 
-        if result_filename.suffix == ".csv":
-            print(colorify(
-                "WARNING: CSV files are deprecated."
-                "Please use Parquet files instead.",
-                YELLOW
-            ))
+        if isinstance(result_filename, list):
+            if any(fname.suffix == ".csv" for fname in result_filename):
+                print(colorify(
+                    "WARNING: CSV files are deprecated."
+                    "Please use Parquet files instead.",
+                    YELLOW
+                ))
+        else:
+            if result_filename.suffix == ".csv":
+                print(colorify(
+                    "WARNING: CSV files are deprecated."
+                    "Please use Parquet files instead.",
+                    YELLOW
+                ))
```

Comment:
That's a nice catch indeed @MatDag!

For conciseness I suggest this reformulation:
```python
if isinstance(result_filename, list):
    is_csv_file = any(fname.suffix == ".csv"
                      for fname in result_filename)
else:
    is_csv_file = result_filename.suffix == ".csv"

if is_csv_file:
    print(colorify(
        "WARNING: CSV files are deprecated."
        "Please use Parquet files instead.",
        YELLOW
    ))